qemu/tests/cfg/fwcfg.cfg

```diff
@@ -0,0 +1,29 @@
+- fwcfg:
+    only Windows
+    type = fwcfg
+    driver_name = "fwcfg"
+    required_qemu = [6.0.0, )
+    vmcoreinfo = yes
+    start_vm = no
+    tmp_dir = "/var/tmp"
+    get_avail_disk = df -h /home | grep home | awk -F "G" '{print $3}'
+    unzip_cmd = 'powershell -command "Expand-Archive %s:\Memory.dmp.zip %s:"'
+    unzip_speed = 60
+    move_cmd = "move {0}:\*Memory.dmp {0}:\Memory.dmp"
+    chk_id_cmd = 'type %s | find /i "LIVE_SYSTEM_DUMP (161)"'
+    images += " stg"
+    image_name_stg = "images/storage"
+    remove_image_stg = yes
+    no ppc64 ppc64le
```

Comment:
What about aarch64?

Reply:
Hi @vivianQizhu  Confirm with feature owner before, aarch64 is support vmcoreinfo device, only power pc does not support now~

Reply:
@peixiu So this test case works well on `aarch64` platform? Have you ever tested it or do we need to request them to help review and test?

Reply:
@vivianQizhu Updated as our discussed, thanks~